$MirOS: ports/www/lynx/patches/patch-src_HTML_c,v 1.1 2011/02/22 20:43:13 tg Exp $

	#1, #2, #3, #4, #5, #6, #8: Fix spelling; suggested for upstream.
	#7: Avoid making read-only input fields

--- src/HTML.c.orig	Mon Feb 13 02:45:53 2012
+++ src/HTML.c	Sat Mar  3 18:20:22 2012
@@ -397,7 +397,7 @@ void HTML_put_character(HTStructured * m
     case HTML_PRE:		/* Formatted text */
 	/*
 	 * We guarantee that the style is up-to-date in begin_litteral. But we
-	 * still want to strip \r's.
+	 * still want to strip \rs.
 	 */
 	if (c != '\r' &&
 	    !(c == '\n' && me->inLABEL && !me->inP) &&
@@ -414,7 +414,7 @@ void HTML_put_character(HTStructured * m
     case HTML_PLAINTEXT:
 	/*
 	 * We guarantee that the style is up-to-date in begin_litteral.  But we
-	 * still want to strip \r's.
+	 * still want to strip \rs.
 	 */
 	if (c != '\r') {
 	    me->inP = TRUE;
@@ -486,7 +486,7 @@ void HTML_put_character(HTStructured * m
 	HText_setLastChar(me->text, ' ');	/* set it to a generic separator */
 
 	/*
-	 * \r's are ignored.  In order to keep collapsing spaces correctly we
+	 * \rs are ignored.  In order to keep collapsing spaces correctly we
 	 * must default back to the previous separator if there was one.
 	 */
     } else if (c == '\r' && HText_getLastChar(me->text) == ' ') {
@@ -637,7 +637,7 @@ void HTML_put_string(HTStructured * me, 
 		} else if (c == '\r' &&
 			   HText_getLastChar(me->text) == ' ') {
 		    /*
-		     * \r's are ignored.  In order to keep collapsing spaces
+		     * \rs are ignored.  In order to keep collapsing spaces
 		     * correctly, we must default back to the previous
 		     * separator, if there was one.  So we set LastChar to a
 		     * generic separator.
@@ -1968,7 +1968,7 @@ static int HTML_start_element(HTStructur
 	UPDATE_STYLE;
 	CHECK_ID(HTML_GEN_ID);
 	/* Add a \r (new line) if these three conditions are true:
-	 *   1. We are not collapsing BR's, and
+	 *   1. We are not collapsing BRs, and
 	 *   2. The previous line has text on it, or
 	 *   3. This line has text on it.
 	 * Otherwise, don't do anything. -DH 980814, TD 980827
@@ -3529,7 +3529,7 @@ static int HTML_start_element(HTStructur
 	    /*
 	     * The MAP must be in the current stream, even if it had a BASE
 	     * tag, so we'll use its address here, but still use the BASE, if
-	     * present, when resolving the AREA elements in it's content,
+	     * present, when resolving the AREA elements in its content,
 	     * unless the AREA's HREF is a lone fragment and
 	     * LYSeekFragAREAinCur is set.  - FM && KW
 	     */
@@ -4781,8 +4781,10 @@ static int HTML_start_element(HTStructur
 		 */
 		I.value = ImageSrc;
 	    }
+#if 0
 	    if (present && present[HTML_INPUT_READONLY])
 		I.readonly = YES;
+#endif
 	    if (present && present[HTML_INPUT_CHECKED])
 		I.checked = YES;
 	    if (present && present[HTML_INPUT_SIZE] &&
@@ -5030,7 +5032,7 @@ static int HTML_start_element(HTStructur
 	LimitValue(me->textarea_rows, MAX_TEXTAREA_ROWS);
 
 	/*
-	 * Lynx treats disabled and readonly textarea's the same -
+	 * Lynx treats disabled and readonly textareas the same -
 	 * unmodifiable in either case.
 	 */
 	me->textarea_readonly = NO;
